packages/webapp/src/components/Map/LineMapBoxes/index.jsx

```diff
@@ -39,6 +41,8 @@ export default function PureLineBox({
   });
 
   const { t } = useTranslation();
+  const measurmentUnit = useSelector(measurementSelector);
+  const maxValue = measurmentUnit == 'metric' ? 1600 : 5280;
```

Comment:
This should be `===`. Also, I would put this in a `useMemo` like `const maxValue = useMemo(() => measurmentUnit === 'metric' ? 1600 : 5280, [measurmentUnit]);`. Otherwise this value will be recalculated everytime any state changes.

Reply:
Also measurement is spelled wrong here, missing an 'e'